### PR TITLE
Update K8s "Hello World" - python

### PR DESCRIPTION
**Pull request by kelsk**

### CODE REVIEW

File: python/python-hello-world/src/app.py [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
-    message = "Hello World"
-    return message
+    message = "It's running!"
 
-if __name__ == '__main__':
-    server_port = os.environ.get('PORT')
-    if server_port is None:
-        print("error: PORT environment variable not set")
-        exit(1)
+    return render_template('index.html', message=message)
```
Comment (dinagraves) [inline python/python-hello-world/src/app.py:32]: You're passing in a message to your template, but your template doesn't seem to have any variables to interact with this message?  

Do we need the template?

File: python/python-hello-world/src/app.py [hunk 2]
```diff
@@ -1,22 +1,36 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 """
 A sample Hello World server.
 """
 import os
-from flask import Flask
+import requests
```
Comment (dinagraves) [inline python/python-hello-world/src/app.py:19]: You don't seem to be using requests?   A Python linter would catch unused imports.

**Review by simonz130 — APPROVED** [review]

**Pull request merged**